cliquet/tests/test_views_hello.py

```diff
@@ -53,8 +53,8 @@ def test_public_settings_can_be_set_with_and_without_prefix(self):
         response = self.app.get('/')
         settings = response.json['settings']
         self.assertIn('cliquet.paginate_by', settings)
-        self.assertIn('myapp.paginate_by', settings)
-        self.assertIn('myapp.project_version', settings)
+        self.assertIn('paginate_by', settings)
```

Comment:
Because the project_name is explicit for ``myapp.paginate_by`` we should probably allow it to be prefixed as we do for cliquet.